R/conf_mat.R

```diff
@@ -102,23 +102,40 @@ conf_mat.data.frame <- function(data,
     warn_conf_mat_dots_deprecated()
   }
 
-  names <- names(data)
+  truth <- enquo(truth)
+  estimate <- enquo(estimate)
+  case_weights <- enquo(case_weights)
 
-  truth <- tidyselect::vars_pull(names, {{truth}})
+  truth <- yardstick_eval_select(
+    expr = truth,
+    data = data,
+    arg = "truth",
+    error_call = error_call
```

Comment:
I don't think you have something called `error_call` here?

Reply:
You can probably just remove this line and let the default arg do its job

Reply:
If you add a test where tidyselection fails (like if the column doesnt exist in the data) then you'd catch the bug here